test/util/fake-timers-test.js

```diff
@@ -414,6 +416,8 @@ describe("sinon.clock", function () {
         });
 
         it("passes 2 hours, 34 minutes and 12 seconds", function () {
+            this.timeout(50000);
```

Comment:
I agree. The problem from a quick glance seems to be based on the implementation of [`clock.tick` in lolex](https://github.com/sinonjs/lolex/blob/master/src/lolex-src.js#L479-L524). Here is [some (very little) context](https://github.com/sinonjs/sinon/pull/1043#issuecomment-222141239). I haven't put a lot of time into investigating though.